src/style.css

```diff
@@ -1,3 +1,110 @@
+:root {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  --link-color: #00acdc;
+  --text-regular-color: #616070;
+  --text-bold-color: #060b35;
+}
+
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+}
+
+.preview {
+  margin: auto;
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.title {
+  margin-top: 40px;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-bold-color);
+}
+
+.code {
+  margin-bottom: 16px;
+  color: var(--text-regular-color);
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+  color: var(--text-bold-color);
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars :nth-child(-n+4) {
```

Comment:
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.